controllers/pwa.js

```diff
@@ -22,51 +22,46 @@ const lighthouseLib = require('../lib/lighthouse');
 const Pwa = require('../models/pwa');
 const router = express.Router(); // eslint-disable-line new-cap
 const libMetadata = require('../lib/metadata');
+const cache = require('../lib/data-cache');
 
 const LIST_PAGE_SIZE = 32;
 const DEFAULT_PAGE_NUMBER = 1;
 const DEFAULT_SORT_ORDER = 'newest';
 
+const CACHE_LIFETIME = 60 * 60 * 24; // 24 hours
```

Comment:
This looks too aggressive, specially for last updated and top score pages. We may want to have different timeouts for PWA and last updated pages.